src/main/java/core/basesyntax/impl/SupplyHandlerImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class SupplyHandlerImpl implements OperationHandler {
```

Comment:
The class `SupplyHandlerImpl` is missing javadoc. It is a good practice to add a javadoc to explain the purpose of the class and its responsibilities.